src/modules/answer/answer.controller.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Controller,
+  Delete,
+  Get,
+  HttpStatus,
+  Post,
+  Put, Query, UsePipes,
+} from '@nestjs/common';
+import {
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { AnswerService } from './answer.service';
+import { ROUTES } from '../../shared/config/routes';
+import { AnswerDto } from '../database/dto/answer.dto';
+import { ValidationPipe } from '../../shared/pipes/validation.pipe';
+import { GetByIdDto } from '../../shared/dto/get-by-id.dto';
+import { UpdateAnswerDto } from './dto/received/update-answer.dto';
+import { CreateAnswerDto } from './dto/received/create-answer.dto';
+
+@ApiTags(ROUTES.ANSWER.MAIN)
+@Controller(ROUTES.ANSWER.MAIN)
+export class AnswerController {
+  constructor(private readonly answerService: AnswerService) {}
+
+  @Get()
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Answers was found',
+    type: AnswerDto,
+  })
+  @UsePipes(new ValidationPipe())
+  getAnswers(): Promise<AnswerDto[]> {
+    return this.answerService.getAnswers();
```

Comment:
do you need to get all answers for app? or only for single article? if for article, add article id + validation